frontend/javascripts/dashboard/advanced_dataset/dataset_table.tsx

```diff
@@ -411,6 +447,9 @@ class DatasetTable extends React.PureComponent<Props, State> {
   // rendering). That's why it's not included in this.state (also it
   // would lead to infinite loops, too).
   currentPageData: RowRenderer[] = [];
+  getIsUserAdminOrDatasetManager(): boolean {
+    return this.props.isUserAdmin || this.props.isUserDatasetManager;
+  }
```

Comment:
there already is `isUserAdminOrDatasetManager` in utils.ts. I'd suggest to either pass the result of that accessor as a prop or pass `activeUser` as a prop and use the accessor instead of this helper method.